Tasks/PackerBuildV1/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,1620 @@
+
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
```

Comment:
Did you need to add any new node module dependency? if so, then a third party notice should be added. I think you have not, but just confirming.

Reply:
No new node dependencies added